README.rst

```diff
@@ -65,6 +67,12 @@ Usage
     response = await nova.servers.create(server=specs)
     print(response)
 
+    volume = {"size": 200,
+              "imageRef": "image_id",
+              "name": "some_name"}
+
+    response = await cinder_client.volumes.create(volume=volume)
```

Comment:
Just to make this example more copy-paste friendly, I would fix this typo `cinder_client` -> `cinder`

Reply:
Fixed! My copy & paste mistake. ;-)